addons/point_of_sale/static/src/app/navbar/navbar.js

```diff
@@ -47,7 +47,26 @@ export class Navbar extends Component {
         onMounted(async () => {
             this.isSystemUser = await user.hasGroup("base.group_system");
         });
+        useExternalListener(document, "keydown", this.handleKeydown.bind(this));
     }
+
+    handleKeydown(event) {
+        if (
+            this.inputRef.el &&
```

Comment:
here you have to check that the input is actually rendered. Typing a key on the floor screen in restaurant for ex results in an error